docs/source/tutorials/token_classification.ipynb

```diff
@@ -0,0 +1,730 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "d0d2e007",
+   "metadata": {},
+   "source": [
+    "# Finding Label Errors in Token Classification Datasets \n",
+    "\n",
+    "This tutorial shows how you can use cleanlab to find potential label errors in text datasets used for the NLP task of token classification. In token-classification, our data consists of a bunch of sentences (aka documents) in which every token (aka word) is labeled with one of K classes, and we train models to predict the class of each token in a new sentence. Example applications include part-of-speech-tagging or entity recognition, which is the focus on this tutorial. Here, we use CONLL-2003 named entity recognition dataset which contains 20,718 sentences and 301,361 tokens, where each token is labeled with one of the following classes:\n",
+    "\n",
+    "- LOC (location entity)\n",
+    "- PER (person entity)\n",
+    "- ORG (organization entity)\n",
+    "- MISC (miscellaneous other type of entity)\n",
+    "- O (other type of word that does not correspond to an entity)\n",
+    "\n",
+    "**Overview of what we'll do in this tutorial:** \n",
+    "- Identify potential token label issues using cleanlab's `token_classification.filter.find_label_issues` method. \n",
+    "- Rank sentences using cleanlab's `token_classification.rank.get_label_quality_scores` method. \n",
+    "- TODO: (Clean Learning) Train a more robust model by removing problematic sentences. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1da020bc",
+   "metadata": {},
+   "source": [
+    "## 1. Install the required dependencies \n",
+    "\n",
+    "You can use `pip` to install all packages required for this tutorial as follows: \n",
+    "\n",
+    "    !pip install cleanlab "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ae8a08e0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!wget -nc https://data.deepai.org/conll2003.zip && mkdir data \n",
```

Comment:
[nit] this is technically not installing dependencies but loading data.
I think it might also be helpful to briefly mention the `wget` install or how to download the data manually. See #2 of [audio tutorial](https://github.com/cleanlab/cleanlab/blob/master/docs/source/tutorials/audio.ipynb) for reference